src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/actions/GenerateTestsActionLine.kt

```diff
@@ -32,7 +32,10 @@ class GenerateTestsActionLine : AnAction() {
 
         logger.info("Selected line is $selectedLine")
 
-        evoSuiteRunner.forLine(selectedLine).runTestGeneration()
+        evoSuiteRunner
+            .forLine(selectedLine)
+            .withCacheLines(selectedLine - 1, selectedLine - 1)
```

Comment:
Why is it necessary to have `-1` here?

Reply:
Because intellij uses 0-indexed lines while evosuite uses 1-indexed lines

Reply:
Thanks!